### ISSUE TITLE
Zooming in on modal breaks the cancel button

### ISSUE THREAD

**Issue by mahmoud-hosny92** [siemens/ix#897]
### What happened?

We have IxModal with cancel button. The cancel button is misplaced if we zoom in while the modal is open.
![image](https://github.com/siemens/ix/assets/147726966/3c60e53f-0912-43e5-aa3c-923d7b320631)

The view shouldn't break regardless of the zoom level.

### What type of frontend frameware are you seeing the problem on?

Angular

### Which version of iX do you use?

2.0.3

### Code to produce this issue.

```shell
1- Create an IxModal.
2- Add a cancel button to it.
3- Zoom in so the modal take the whole screen.
```

**Comment by matthiashader** [comment]
Could you provide the code, i just can't reproduce the behavior

**Comment by mahmoud-hosny92** [comment]
This is an example:
![image](https://github.com/siemens/ix/assets/147726966/8f8c5b0f-ff31-42d4-998c-57c8cf72e801)

**Comment by matthiashader** [comment]
Tracked by [IX-861]

**Comment by matthiashader** [comment]
Fixed by #1281.
